### PR TITLE
Hover-tooltips for xrefs in docs

### PR DESCRIPTION
**Pull request by Zac-HD**
Via https://sphinx-hoverxref.readthedocs.io/en/latest/index.html

(hopefully this also does readthedocs preview builds for the PR...)

### CODE REVIEW

File: hypothesis-python/src/hypothesis/_settings.py [hunk 1]
```diff
@@ -379,13 +379,15 @@ def _max_examples_validator(x):
...
-based on the hypothesis to falsify, which will be consistent across
-multiple runs. This has the advantage that it will eliminate any
-randomness from your tests, which may be preferable for some situations.
-It does have the disadvantage of making your tests less likely to
-find novel breakages.
+If True, seed Hypothesis' random number generator using a hash of the test
+function, so that every run will test the same set of examples until you
+update Hypothesis, Python, or the test function.
+
+This allows you to `checking for regressions and looki for bugs
```
Comment (DRMacIver) [inline hypothesis-python/src/hypothesis/_settings.py:386]: "looki"?

**Comment by Zac-HD** [comment]
Yes, this is basically my list of small tweaks to the docs, and it could be split into two PRs. I'll also look into the link problem.

If you have time for some reviews though, I'd prefer to get the example reporting in to unblock some Numpy work, typing extensions to ease work on a hypothesmith paper, and of course the ghostwriter (scipy conference is in two weeks and I'd really like to present it then). This docs stuff is relatively low priority!

**Comment by DRMacIver** [comment]
> If you have time for some reviews though

The constraint isn't really time, it's mental energy / focus. This was an easy thing to review.

**Comment by Zac-HD** [comment]
@DRMacIver - the 404 bug will be fixed by https://github.com/readthedocs/sphinx-hoverxref/pull/82; happy to either merge before that or after as you prefer.

**Comment by Zac-HD** [comment]
@DRMacIver - the link bug has been fixed; I've bumped our pinned dependency to the new version and confirmed that links in tooltips now work.  Anything else to do, or are we ready to merge?

**Comment by Zac-HD** [comment]
Ping @HypothesisWorks/hypothesis-python-contributors?  You can see how this works on the docs preview, e.g. at https://hypothesis--2467.org.readthedocs.build/en/2467/data.html#hypothesis.strategies.data

**Pull request merged**